src/Microsoft.Health.Fhir.Core/Features/Search/SearchService.cs

```diff
@@ -232,6 +232,16 @@ protected SearchService(ISearchOptionsFactory searchOptionsFactory, IFhirDataSto
             throw new NotImplementedException();
         }
 
+        public virtual Task<IReadOnlyList<(long start, long end, long globalStart, long globalEnd)>> GetSurrogateIdRanges(
```

Comment:
I think the naming standards now prefer pascal-case field names on tuples. Though this might also be a good case for returning a `record`.

```suggestion
        public virtual Task<IReadOnlyList<(long Start, long End, long GlobalStart, long GlobalEnd)>> GetSurrogateIdRanges(
```

Reply:
Sure.